src/main/java/uk/gov/hmcts/darts/audio/component/impl/AnnotationXmlGeneratorImpl.java

```diff
@@ -21,7 +23,7 @@
 @Slf4j
 public class AnnotationXmlGeneratorImpl extends AbstractDocumentGenerator {
 
-    private static final String ANNOTATION_FILENAME = "annotations.xml";
+    private static final ZoneId EUROPE_LONDON_ZONE = ZoneId.of("Europe/London");
```

Comment:
I'm wondering we should all be using the same time constants and so this should be in common perhaps?

Reply:
Updated in commit e339bbc - add DateConverters (similar to https://github.com/hmcts/darts-gateway/pull/90/files#r1294794974)

BST can be very frustrating!  I came to the conclusion today that using `ZonedDateTime` to present date and time to users correctly in BST was needed in darts-api only for the ATS Download Zip metadata files.

To store data and time, we use `OffsetDateTime` in UTC and to present date and time to users, we should use the `ZonedDateTime` for the legacy interfaces. According to the restful api standards for the darts-api we expect request / response for date-time strings in JSON to be in UTC.

See also [5. Common format for timestamps](https://tools.hmcts.net/confluence/display/DMP/5.+Common+format+for+timestamps?src=contextnavpagetreemode) - for initial decision on choosing types for our database / JPA